src/app/components/Field/__snapshots__/Field.test.tsx.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Field clicking a card selects it 1`] = `"translateX(0px) translateY(-25px) scale(1.25)"`;
+
+exports[`Field clicking a card selects it 2`] = `""`;
```

Comment:
are these snapshots with empty strings expected? i can't really tell based on the test names (which is part of my gut hesitation with snapshotting transform values.. i don't know what's expected now or if they change later)

Reply:
i get what you're saying and i'm on board with it. i have a sort of forced reaction to question every snapshot i see because they're easy to update without thinking and i don't really like them, but i think you make a good case for using them here.

also, tiny story/fun-fact, i once replaced a ton of snapshot tests with static expects and the snapshots performed better somehow 😱. i still don't like them, but there's some real magic happening behind the scenes